docs/ilm.asciidoc

```diff
@@ -0,0 +1,190 @@
+[[ilm]]
+[role="xpack"]
+== Index lifecycle management (ILM)
+
+Use the {ref}/getting-started-index-lifecycle-management.html[index lifecycle management]
+feature in {es} to manage your APM Server indices as they age.
+For example, instead of creating daily indices where index size can vary based on the number of agents and number of events sent,
+use an index lifecycle policy to automate a rollover to a new index when the existing index reaches a specified size or age.
```

Comment:
While technically proper, we do use daily rollover without defining a `delete` phase in the current state. You describe later on _why_ we chose this policy, but reading this statement might still be confusing at the current stage.

Reply:
fixed